.github/workflows/lint.yml

```
@@ -0,0 +1,91 @@
+        uv sync --all-extras --dev
+        uv add --dev radon xenon
+    
+    - name: Run Radon complexity analysis
+      run: |
+        uv run radon cc . --min B
+      continue-on-error: true
+    
+    - name: Run Xenon complexity analysis
+      run: |
+        uv run xenon --max-absolute B --max-modules A --max-average A .
+      continue-on-error: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)